client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -129,6 +132,19 @@ export function getEnhancedTasks(
 						},
 					};
 					break;
+				case 'unlock_styles':
+					if ( ! isEnabled( 'limit-global-styles' ) || ! globalStylesInUse ) {
+						return;
+					}
+					taskData = {
+						title: translate( 'Activate advanced design customization tools' ),
+						isCompleted: ! shouldLimitGlobalStyles,
```

Comment:
I don't think it's semantically correct to tie the completeness of the task to if we should limit global styles.